app/boards/shields/cradio/cradio_left.conf

```diff
@@ -1,2 +1,2 @@
-# Copyright (c) 2020 The ZMK Contributors
-# SPDX-License-Identifier: MIT
+CONFIG_ZMK_SLEEP=y
```

Comment:
We'll be turning sleep on by default when ready, individual shields shouldn't be doing so yet.